projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -327,6 +371,12 @@ interface ExplorerContextScope {
   scopeQueryParam: ScopeQueryParam;
 }
 
+type contextMap = {
```

Comment:
PascalCase for type names (also, we generally use interfaces if constructing a brand new type that's not projected from another.

Naming here is also an issue. I think context is an overloaded term, and even knowing the goal of this PR it took me reading the object using this to understand what this was. Maybe something like
``` ts
type AttributeTranslationDictionary = { [key: string]: string; }

const spanToApiTraceAttributeTranslationDictionary = {...};
const apiTraceToSpanAttributeTranslationDictionary = {...};
```